nuxt/api/tracks.ts

```diff
@@ -0,0 +1,86 @@
+import { NuxtAxiosInstance } from '@nuxtjs/axios';
+import {
+  PersistedEntity,
+  TimestampedEntity,
+  createParams,
+  useIncludes,
+  PaginatedResponse,
+  usePagination,
+  PaginationOptions,
+} from '@/api/common';
+import { Reciter } from '@/api/reciters';
+import { Album } from '@/api/albums';
+
+/*
+ * Entity Definitions
+ */
+export interface Track extends PersistedEntity, TimestampedEntity {
+  reciterId: string;
+  reciter?: Reciter;
+  albumId: string;
+  album?: Album;
+  title: string;
+  slug: string;
+  audio: string;
+  lyrics: string;
```

Comment:
`lyrics` and `audio` aren't part of the API response for track by default. They're optional based on `includes`. So this isn't accurate.

Look at the response here and build an accurate interface: https://api.nawhas.test/v1/reciters/tejani-brothers/albums/2019/tracks